Bottom: fd757ff2ef7cf8485c3e558d91dc74afaaec9617
Top:    4c8c6571cebb22afa0c41def0cb18f5cce3421dc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:39:17 +0000

Refresh of add-polygon-parentage-tree

---

diff --git a/src/polygon.c b/src/polygon.c
index be4fcf2..5305cdf 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1304,19 +1304,15 @@ clearPoly (DataType *Data, LayerType *Layer, PolygonType * polygon,
           r_search (layer->line_tree, &region, NULL, line_sub_callback,
                     &info);
         subtract_accumulated (&info, polygon);
-#if 0
         r +=
           r_search (layer->arc_tree, &region, NULL, arc_sub_callback, &info);
 	r +=
           r_search (layer->text_tree, &region, NULL, text_sub_callback, &info);
-#endif
       }
       END_LOOP;
-#if 0
       r += r_search (Data->via_tree, &region, NULL, pin_sub_callback, &info);
       r += r_search (Data->pin_tree, &region, NULL, pin_sub_callback, &info);
       subtract_accumulated (&info, polygon);
-#endif
     }
   polygon->NoHolesValid = 0;
   return r;
